pilot/docker/Dockerfile.proxytproxy

```diff
@@ -1,5 +1,8 @@
-# hadolint ignore=DL3006
-FROM istionightly/base_debug
+# ISTIO_VERSION is used to specify the version of the release
+ARG ISTIO_VERSION=latest
+
+# The following section is used as base image if BASE_DISTRIBUTION=default
+FROM docker.io/sdake/base_debug:${ISTIO_VERSION} as default
 ARG proxy_version
 ARG istio_version
 
```

Comment:
Can't we do `ENV ISTIO_VERSION istio_version` instead of passing another build argument with the same value?

Reply:
yes, I am likley to consolidate  the 2 or 3 places where istio_version is present, and just ue ISTIO_VERSION.

IMO `ARGS` and environment variables should be capitalized.